aries-site/src/pages/components/button.mdx

```diff
@@ -49,6 +49,26 @@ The HTML elements for buttons and links (a.k.a. anchors) describe a very specifi
 - Use buttons when you are performing an action, such as: "submit," "merge," "create new," "upload," etc.
 - An action almost always occurs on the same page
 
+### Toggle Buttons
+
+Toggle buttons are used either to switch between two states, such as check and unchecked, or to trigger two actions like on or off.
+
+#### Toggle Buttons with Icons
+
+Icon buttons have an aria-label that will describe the action that the user will be performing when using screen readers.
```

Comment:
For consistency with the text below:

 "... have an `a11y-title`, `aria-label`, or button `label` that describes the action ..."